components/public-api/typescript-common/src/public-api-converter.ts

```diff
@@ -249,6 +212,127 @@ export class PublicAPIConverter {
         return spec;
     }
 
+    workspaceContextToWorkspaceInitializer(arg: WorkspaceContext, cloneUrl?: string): WorkspaceInitializer {
+        const result = new WorkspaceInitializer();
+        if (OpenPrebuildContext.is(arg)) {
+            result.specs = [
+                new WorkspaceInitializer_Spec({
+                    spec: {
+                        case: "prebuild",
+                        value: new PrebuildInitializer({
+                            prebuildId: arg.openPrebuildID,
+                        }),
+                    },
+                }),
+            ];
+            if (cloneUrl) {
+                result.specs.push(
+                    new WorkspaceInitializer_Spec({
+                        spec: {
+                            case: "git",
+                            value: new GitInitializer({
```

Comment:
We need clone URL (it's not that match current dashboard, we're using something like `open-prebuild/<prebuild_id>/<clone_url>`,  `clone_url` could be PR url or something else. But we can't do it the same

Reply:
I don't think we need this case, we need to change the dashboard to use GitInitializer.remoteUri (it is cloneUrl)